lib/gui/app.js

```diff
@@ -185,20 +185,15 @@ app.run(() => {
 
     const currentFlashState = flashState.getFlashState()
 
-    // There is usually a short time period between the `isFlashing()`
+    // NOTE: There is usually a short time period between the `isFlashing()`
     // property being set, and the flashing actually starting, which
     // might cause some non-sense flashing state logs including
     // `undefined` values.
-    //
-    // We use the presence of `.eta` to determine that the actual
-    // writing started.
-    if (!currentFlashState.eta) {
```

Comment:
As I mentioned in flowdock, I believe this was a simple logic error and the `if (!currentFlashState.eta)` here should have been `if (currentFlashState.eta)`. I've just tested that locally, and it brings back the progress-spam to DevTools :wink:

Reply:
Yeah, but eta can be zero (if the speed is zero), and we'll want to have that context in case of a bug, hence that check is removed completely here.

Reply:
Ah, okay.